zebedee-cms/src/main/java/com/github/onsdigital/zebedee/ZebedeeConfiguration.java

```diff
@@ -126,10 +118,14 @@ public ZebedeeConfiguration(Path rootPath, boolean enableVerificationAgent) thro
         this.applicationKeysPath = createDir(zebedeePath, APPLICATION_KEYS);
         this.redirectPath = this.publishedContentPath.resolve(Content.REDIRECT);
         this.servicePath = createDir(zebedeePath, SERVICES);
+        this.keyRingPath = createDir(zebedeePath, KEYRING);
 
         if (!Files.exists(redirectPath)) {
             Files.createFile(redirectPath);
         }
+        if (!Files.exists(keyRingPath)){
+            Files.createFile(keyRingPath);
+        }
```

Comment:
This isn't necessary line 121 
````
this.keyRingPath = createDir(zebedeePath, KEYRING);
````
Is already checking if the dir exists and creating it if not.